packages/lit-html/src/is-server.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @fileoverview
+ *
+ * This file exports a boolean const whose value will depend on what environment
+ * the module is being imported from.
+ */
+
+const NODE_MODE = false;
+
+/**
+ * Boolean value that will `true` when imported via the "node" export condition
+ * and `false` otherwise.
```

Comment:
Maybe add a similar sentence about when you would use it for SSR, from the changeset. Since this is what will show up on lit.dev API docs.